<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InsertHarmonicDialog</class>
 <widget class="QDialog" name="InsertHarmonicDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>259</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Insert Row</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="amplitudeLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Amplitude:</string>
   </property>
  </widget>
  <widget class="QLabel" name="frequencyLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Frequency:</string>
   </property>
  </widget>
  <widget class="QLabel" name="phaseLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Phase:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="functionField">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>271</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Sin</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Cos</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="functionLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Function:</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="amplitudeField">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>271</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-999.000000000000000</double>
   </property>
   <property name="maximum">
    <double>999.000000000000000</double>
   </property>
   <property name="value">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="frequencyField">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>271</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-999.000000000000000</double>
   </property>
   <property name="maximum">
    <double>999.990000000000009</double>
   </property>
   <property name="value">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="phaseField">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>140</y>
     <width>271</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-999.000000000000000</double>
   </property>
   <property name="maximum">
    <double>999.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="harmonicTextView">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>180</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>1 * Sin(1 * x + 0)</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>InsertHarmonicDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>InsertHarmonicDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
